src/esp/gfx/RenderTarget.cpp

```diff
@@ -116,6 +116,8 @@ struct RenderTarget::Impl {
 
   void blitRgbaToDefault() {
     framebuffer_.mapForRead(RgbaBuffer);
+    ASSERT(framebuffer_.viewport() == GL::defaultFramebuffer.viewport());
```

Comment:
:thinking: I think you wanted to do `framebuffer_.viewport().size() == GL::defaultFramebuffer.viewport().size()`, right? Because this will blow up if the offsets are different (which I think they are, in case of stereo rendering).

Reply:
That's right. Let me fix it in a separate PR.